CHANGELOG.md

```diff
@@ -1,81 +0,0 @@
-## Changelog
```

Comment:
@amokhtar I dont know why the changelog is duplicated in so many places. For now, I'm assuming we will just keep this stuff in the root README file. We should remove these other duplicates